airbyte_cdk/sources/declarative/concurrent_declarative_source.py

```diff
@@ -3,7 +3,7 @@
 #
 
 import logging
-from typing import Any, Generic, Iterator, List, Mapping, Optional, Tuple, Union
+from typing import Any, Generic, Iterator, List, Mapping, Optional, Tuple, Union, Callable
```

Comment:
@aaronsteers is it expected that `poetry run ruff format` does not order the imports?

Reply:
Ignore that, it seems like I didn't have the right ruff command